go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/xanzy/go-gitlab
+module github.com/temamagic/go-gitlab
```

Comment:
You should not change this path in a PR.

Reply:
Hm, strange, I'll change this after PR... anyway, thx!